src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -3175,18 +3175,19 @@ namespace PlantCondLoopOperation {
 						UniqueIDName = PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Name;
 						ActuatorType = "On/Off Supervisory";
 						SetupEMSActuator( ActuatorName, UniqueIDName, ActuatorType, Units, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).EMSCtrlOverrideOn, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).EMSCtrlOverrideValue );
-					} else if ( LoopSideNum == DemandSide ) {
+						for ( CompNum = 1; CompNum <= PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).TotalComponents; ++CompNum ) {
+							ActuatorName = "Plant Component " + ccSimPlantEquipTypes( PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).TypeOf_Num );
+							UniqueIDName = PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).Name;
+							ActuatorType = "On/Off Supervisory";
+							SetupEMSActuator( ActuatorName, UniqueIDName, ActuatorType, "[W]", PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).EMSLoadOverrideOn, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).EMSLoadOverrideValue );
+						}
+					}
+					else if ( LoopSideNum == DemandSide ) {
```

Comment:
So the point of moving the EMSActuator setup call into here was to avoid issues such as EMS actuators being applied to both the condenser and evaporator connections of a single chiller?  I'm _OK_ with this, since in pretty much all cases, plant components will be connected in the _traditional E+_ fashion, and you'll be wanting to control it on the supply level.  But just for reference, after the plant re-write of yon' few years back, the plant is pretty loose about hooking things up.  So this not guaranteed to solve that type of problem. Theoretically, you could connect the chiller from a supply side to another supply side as awkward as that would be.  Also, it will restrict if someone wants to actuate a plant component purposely connected on the demand side.  (I'm thinking people playing around with placement of economizers, e.g.).  

@EnergyArchmage Could the EMS magi comment on whether this seems suitable?  As I mentioned, I think this will work because most all plants will be set up the same, standard, way.  But it could be potentially limiting.

I'm pushing up a commit with the white space fix above and making sure develop is merged in.  If all comes back clean, and if the mage is OK with it, it can merge in.